localization/react-intl/src/app/components/cds/media-cards/MediaCount.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "id": "sharedItemCard.medias",
-    "description": "A count of medias in an item. Title-case where applicable. Example: 3 Medias",
-    "defaultMessage": "{mediaCount, plural, one {# Media} other {{mediaCount, number} Medias}}"
```

Comment:
I'm not sure this is a good change as it's probably worth having clear singular/plurals in other languages.

Reply:
@amoedoamorim I wasn't actually sure how to handle this, maybe you can help.
I changed this, because in English Media/Medias as both "correct" plurals, but to most native english speakers Medias sounds wrong to the ear. So I would prefer in english to use Media for both singular or plural, but wasn't sure how to do this for other languages.

Should I just revert this change and put Media for both situations? and then the other languages can be translated for themselves?

Reply:
> Should I just revert this change and put Media for both situations? and then the other languages can be translated for themselves?

Yeah, sounds like the best approach here